components/script/dom/htmltimeelement.rs

```diff
@@ -44,10 +43,7 @@ impl HTMLTimeElementMethods for HTMLTimeElement {
         if element.has_attribute(&local_name!("datetime")) {
             return element.get_string_attribute(&local_name!("datetime"))
         } else {
-            match element.GetInnerHTML() {
-                Ok(x) => x,
-                _ => DOMString::new(),
-            }
+            element.upcast::<Node>().child_text_content()
```

Comment:
Well isn't this funny. Let's see what the spec says:

> The dateTime IDL attribute must reflect the element's datetime content attribute.

Nothing in there about the text content. There is something about it in <https://html.spec.whatwg.org/multipage/semantics.html#dom-time-datetime>, but that's something else entirely, and I'm not sure it's exposed to JS.

Let's revert everything related to the time element here, and fix this method separately.